nucleus/__init__.py

```diff
@@ -659,11 +659,16 @@ def dataitem_ref_id(self, dataset_id: str, reference_id: str):
 
     @deprecated("Prefer calling Dataset.predictions_refloc instead.")
     @sanitize_string_args
-    def predictions_ref_id(self, model_run_id: str, ref_id: str):
+    def predictions_ref_id(
+        self, model_run_id: str, ref_id: str, dataset_id: Optional[str] = None
+    ):
+        if dataset_id:
+            raise RuntimeError(
+                "Need to pass a dataset id. Or use Dataset.predictions_refloc."
+            )
```

Comment:
I'm not quite sure how this worked before but I just added a optional dataset_id such that this works here but trips up existing usage and points them to the the non-deprecated method.

Reply:
This is fine :)